search/TopologicalSearch.cpp

```diff
@@ -26,4 +37,24 @@ bool TopologicalSearch::isInCircle(const Coordinates &center, Meters radius, con
     return inCircle;
 }
 
+bool TopologicalSearch::isInCircle(const Coordinates &center, Meters radius, const PointList &entityGeometry) const {
```

Comment:
We can use inside the function instead:
Coordinates closest =entityGeometry.getClosestPoint(center);
return CoordinatesMath::calculateDistance(center, closest) <= radius;

Reply:
Fixed